koroonakaart/src/components/Statsbar.vue

```diff
@@ -80,19 +80,19 @@
     <b-row>
       <b-col class="statsbar-item" md>
         <div class="statsbar-heading">
-          <h5>{{ $t("hospitalised") }}</h5>
+          <h5>{{ $t("hospitalized") }}</h5>
         </div>
-        <h1>{{ hospitalisedNumber | formatNumber(currentLocale) }}</h1>
+        <h1>{{ hospitalizedNumber | formatNumber(currentLocale) }}</h1>
         <h5
           :class="
-            rawHospitalisedChanged === 0
+            rawHospitalizedChanged === 0
```

Comment:
This code can be optimized using computed properties, instead of two-level if-else statement?

Reply:
I'm sure a lot of the frontend logic could be optimized, I tried to leave it unchanged whenever possible for this PR

Reply:
@lietu Thank you for answer! Yes, small changes not releated with optimization isn't so critical.